ElectronClient/gui/ConfigScreen.jsx

```diff
@@ -270,8 +280,22 @@ class ConfigScreenComponent extends React.Component {
 			borderRadius: 4,
 		});
 
+		const detectTheme = (value) => {
+			this.setState({ shouldUseDarkColors: bridge().shouldUseDarkColors() });
+
+			// active theme should change to preferred theme based on system detected theme
+			if (value == true) {
```

Comment:
Please use a more descriptive name for `value`.  Perhaps `autoDetect`

Reply:
Yes you are right I didn't thought of it, I'll change it to `autoDetect`